pycbc/waveform/nr_waveform.py

```diff
@@ -93,6 +91,8 @@ def get_param(value):
     
     # Open NR file:
     fp = h5py.File(hd5_file_name, 'r')
+    # Add on intrinsic phase to provided phi
+    phi += fp.attrs['coa_phase']
```

Comment:
Patricia: Maybe in the file it would be better to call this the reference phase, as this is not the coa_phase? For the HDF5 file, we do not have to be bound by limitations on the XML files.